build_web_compilers/lib/src/web_entrypoint_builder.dart

```diff
@@ -23,10 +23,16 @@ enum WebCompiler {
 
 /// The top level keys supported for the `options` config for the
 /// [WebEntrypointBuilder].
-const _supportedOptions = const [_compiler, _dart2jsArgs, _buildRootAppSummary];
+const _supportedOptions = const [
+  _compiler,
+  _dart2jsArgs,
+  _buildRootAppSummary,
+  _ignoreCastFailures
+];
 const _buildRootAppSummary = 'build_root_app_summary';
 const _compiler = 'compiler';
 const _dart2jsArgs = 'dart2js_args';
+const _ignoreCastFailures = 'ignore_cast_failures';
```

Comment:
Would be nice to have the behavior documented *somewhere*.

Re: DDC-only, likely to change default in the future, etc

Reply:
It's in the changelog - but ya we should get in the habit of updating real docs with new features.